docs/CONTRIBUTING.md

```diff
@@ -38,8 +38,8 @@ See [reporting of security vulnerabilities](/SECURITY).
 ## Posting Pull Requests
 
 * Make sure to post PRs only relative to a recent tip of the `main` branch.
-* Follow our [Coding Style](/CODING_STYLE) when contributing code. This is a requirement for all code we merge.
-* Please make sure to test your change before submitting the PR. See the [Hacking guide](/HACKING) for details on how to do this.
+* Follow our [Coding Style](./CODING_STYLE.md) when contributing code. This is a requirement for all code we merge.
```

Comment:
Why? We need absolute path here, and the render files are .html, not .md ...

If currently the links are broken for systemd.io, that's because the recent changes haven't been deployed.

Reply:
I wasn't aware of https://systemd.io/CONTRIBUTING/ . My experience as a first-time contributor was to open a new PR, get side dialog from Github telling me to review the [contributing guide](/docs/CONTRIBUTING.md) and get a 404 when checking those files.

Is there a way to have the links work both on Github and on systemd.io ? I see a few absolute links (e.g. the README one) and was wondering why the difference, that makes sense now.

Reply:
@Geod24 [FYI](https://github.com/systemd/systemd/pull/32324#issuecomment-2066851118)

Reply:
> Is there a way to have the links work both on Github and on systemd.io ?

Well, I'd rather avoid reading raw .md files (although they might be readable). Instead, convert links that point to these markdown files to point to systemd.io?

Reply:
Good point. Changed to that.

Reply:
Hmm, I don't see it?

Reply:
So, I sympathize with having valid links in the documents that are rendered by GH as well, and I think I have a potential solution, but will need to do a couple of tests first just to make sure it doesn't break anything.

Reply:
@YHNdnzj : I amended the commit but forgot to add 🤦 Fixed now.

@mrc0mmand : I don't have a strong opinion either way. The docs on systemd.io are nicely rendered, but I could live with the ones in Github as well. What matters most to me is that first time contributors don't run into obstacles when getting started. Perhaps there's value in having them stay on Github but I leave that judgement call to more seasoned contributors.

Reply:
Oh well, nevermind. The trailing slash we have in the page permalink is really annoying and thwarts any attempts to work around this, but we can't drop it without breaking existing links. Not sure what to do here, but rI guess eplacing the absolute links with fully qualified URLs might be a possibility.

Reply:
/cc @keszybz